main.py

```diff
@@ -2,6 +2,31 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    first_name = ["George", "Kianu", "Scarlett", "Will", "Jaden", "Scarlett"]
+    last_name = ["Klooney", "Reaves", "Keegan", "Smith", "Smith", "Johansson"]
+
+    for first, last in zip(first_name, last_name):
```

Comment:
why not create a list of tuples? you will not need to use zip() later
Also naming `first` and `last` is not very informative